packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -277,6 +277,7 @@ The format for conditional expressions is
 * The `OR` operator
 * The `!` operator to designate NOT
 * Standard relational operators including <code>&lt;</code>, <code>&gt;</code>, <code>&lt;=</code>, and <code>&gt;=</code>.
+* The `matches` operator to evaluate a String against a regular expression.
```

Comment:
Do you know when we shipped this operator?

Reply:
Not sure.  I have verified that 2021.03.1 has it.

Reply:
Do we know why this change was not out with the feature release when the new operator was released?

Reply:
I suspect this has been in the product for some time since EL is based on Spring but just wasn't documented.

Reply:
Ok. I just felt that we should always update the doc when we release a feature.